src/openrct2/scenario/scenario.h

```diff
@@ -368,7 +381,7 @@ extern rct_s6_info gS6Info;
 extern char gScenarioName[64];
 extern char gScenarioDetails[256];
 extern char gScenarioCompletedBy[32];
-extern char gScenarioSavePath[MAX_PATH];
+extern char gScenarioSavePath[260];
```

Comment:
I completely agree, but platform.h is just too big and bulky to include. Consider a new header file containing the limits or use `common.h`.